playground/component/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "component"
+edition = "2021"
+
```

Comment:
Might be worth adding a `publish = false` here, and as part of the main workspace thisi can also use `edition.workspace = true` along with `[lints] workspace = true` too

Reply:
Done.

Reply:
Hm, actually, adding `[lints] workspace = true` [prevents building](https://github.com/bytecodealliance/wasm-tools/actions/runs/9433506842/job/25984523557) because the lints don't like the code that `cargo component` generates. I've removed it for the moment; happy to take other suggestions.

Reply:
Ah ok no worries, I can try to work on that once this lands.